.github/workflows/helm-chart.yml

```diff
@@ -19,3 +19,36 @@ jobs:
         run: nix-shell --run ".ci/helm-docs.sh"
       - name: HelmChart Lint
         run: nix-shell --run ".ci/helm-lint.sh"
+
+  publish:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    needs: ["chart-test"]
+    if: ${{ github.event_name != 'pull_request' }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+          fetch-depth: 0
+    - uses: DeterminateSystems/nix-installer-action@v14
+    - name: Pre-populate nix-shell
+      run: |
+        export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' nix/sources.json -r)
+        echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
+        nix-shell --run "echo"
+
+    - name: Configure Git
+      run: |
+        git config user.name "$GITHUB_ACTOR"
+        git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+    - name: Run chart-releaser
```

Comment:
How are the container images published?
Do you think we can publish this soon?

Reply:
We are ready to publish right now, but I think first we should change K8s client library and fix currently opened issues

Reply:
Can't we start publishing before that? I think we can at least start publishing the chart so it's easier for folks to start playing with it more or is there anything that would be a cause for concern?

Reply:
Alright, should I pick this PR and complete it since you're busy with the fs refactoring?